prometrics_test.go

```diff
@@ -100,3 +100,74 @@ func TestCommandFactory(t *testing.T) {
 		time.Sleep(1 * time.Hour)
 	}
 }
+
+
+func TestCommandFactoryWithNameSpace(t *testing.T) {
```

Comment:
This test can't test namespace. 

I think you don't need to add a new test for this.